### PR TITLE
added feedback form

### PR DESCRIPTION
**Pull request by athompsonScottLogic**

### CODE REVIEW

File: whichone/static/js/form.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+$(document).ready(function () {
+    $('#go-to-play').on('click', function (event) {
+        window.location.href = 'play';
+        console.log("yes");
```
Comment (connor-settle) [inline whichone/static/js/form.js]: Remove dev logs from source control

File: whichone/static/js/main.js [hunk 2]
```diff
@@ -44,6 +44,14 @@ $(document).ready(function () {
         }
     });
 
+    $('#feedback-icon').on('click', function (event) {
+        window.location.href = 'form';
+    });
+
+    $('#go-to-play').on('click', function (event) {
+        window.location.href = 'play';
+    });
```
Comment (connor-settle) [inline whichone/static/js/main.js]: Are you able to use `anchor` elements to do the same thing?

File: whichone/templates/play.html [hunk 3]
```diff
@@ -42,6 +42,9 @@
     <div class="bgimg-1">
         <div class="container">
             <div class="nav-bar">
+                    <input type = "image" id="feedback-icon" alt="feedback icon" style="vertical-align:middle;margin:14px 6px"
```
Comment (connor-settle) [inline whichone/templates/play.html]: Its worth organising your styles consistently

**Review by connor-settle — COMMENTED** [review]
Look good 👍

File: whichone/static/js/form.js (hunk 1 above)
Comment (athompsonScottLogic) [reply whichone/static/js/form.js]: sorted

File: whichone/templates/play.html (hunk 3 above)
Comment (athompsonScottLogic) [reply whichone/templates/play.html]: sorted

File: whichone/static/js/main.js (hunk 2 above)
Comment (athompsonScottLogic) [reply whichone/static/js/main.js]: is it better practice? We can change all of our jqueries to anchor elements (I believe), what's the benefit?

Comment (connor-settle) [reply whichone/static/js/main.js]: I guess my thought would be, why pollute JS with template level logic.

If we were using a framework or had some logic around the routing we might need to go through JS. 
The benefit of utilizing DOM features like this is:
* clean JS
* we get the little preview of the link in bottom left hand corner 😄 as a user I prefer to see where a link will take me before I navigate

Comment (athompsonScottLogic) [reply whichone/static/js/main.js]: Thanks for the info! I've now changed it to use anchor elements

File: whichone/static/js/form.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+$(document).ready(function () {
```
Comment (connor-settle) [inline whichone/static/js/form.js]: Looks like this is not required? As I can see the DOM has href on the anchor element.

**Review by connor-settle — APPROVED** [review]

**Comment by PHardySL** [comment]
Few issues from my testing.
**Major:**

- able to send blank forms
- no max character length limit
- limit on number of form submissions

**Minor:**

- "back to play" button needs to look better.
- screen size issues (am sure will be fixed with mobile optimisation integrations
- button overlaps other elements on the screen.

**Low:**

- would be nice to have back button functionality (ease of use)
- perhaps a tool-tip explaining the use of a persons email may be a good idea

File: whichone/static/js/form.js (hunk 4 above)
Comment (athompsonScottLogic) [reply whichone/static/js/form.js]: removed

**Comment by athompsonScottLogic** [comment]
> Few issues from my testing.
> **Major:**
> 
> * able to send blank forms
> * no max character length limit
> * limit on number of form submissions
> 
> **Minor:**
> 
> * "back to play" button needs to look better.
> * screen size issues (am sure will be fixed with mobile optimisation integrations
> * button overlaps other elements on the screen.
> 
> **Low:**
> 
> * would be nice to have back button functionality (ease of use)
> * perhaps a tool-tip explaining the use of a persons email may be a good idea

Tooltip added, minor issues will all be resolved in UI refactor, now has char limit of 300, captcha added to limit spam and spam filter added on Formspree's server side

**Review by connor-settle — APPROVED** [review]

**Pull request merged**